### ISSUE TITLE
Dropdown: Not keyboard accessible

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-core#639]
The dropdown should be keyboard accessible, both in its regular view and when displayed as a panel on small screens.

**Comment by mikewheaton** [comment]
This issue was moved to OfficeDev/office-ui-fabric-js#50
